modApiServer/src/org/aion/api/server/zmq/HdlrZmq.java

```diff
@@ -80,7 +80,7 @@ void getTxWait() {
                 return;
             }
 
-        } catch (Throwable e) {
+        } catch (Exception e) {
             // TODO Auto-generated catch block
             LOGGER.error("zmq takeTxWait failed! " + e.getMessage());
```

Comment:
`getMessage()` may sometime be null, I suggest replacing `" + e.getMessage());` with `", e);`